src/common/etcd_delegate.rs

```diff
@@ -101,7 +106,7 @@ impl EtcdDelegate {
     async fn get_one_kv_async<T: DeserializeOwned, K: Into<Vec<u8>> + Debug + Clone + Send>(
         &self,
         key: K,
-    ) -> DatenLordResult<Option<T>> {
+    ) -> DatenLordResult<Option<(T, KVVersion)>> {
```

Comment:
Maybe we can add a separate function called "get_with_version" because most of the time when we call this interface, we don't need the version. From your pull request, it seems to be causing issues with unused variables and making the code less aesthetically pleasing with added underscores.

Reply:
It will be replace with kvengine, we don't need to waste too much effort on this.

Reply:
And actully most part need to use txn or lock finally.

Reply:
But this modification includes some unnecessary changes in this pull request, such as csi/meta, and causes some aesthetic issues. If you need get_with_version, you just need to create a new function

Reply:
Changed to related functions to fn xxxx and fn xxxx_return_with_version, and the default xxxx returns withouts version.